### PR TITLE
#284 Add variables for SDPi version string (short version number)

### PR DESCRIPTION
**Pull request by d-gregorczyk**
## ☑ Mandatory Tasks

The following aspects have been respected by the pull request assignee and at least one reviewer:
  
- Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [ ] Reviewer

### CODE REVIEW

**Review by JavierEspina — COMMENTED** [review]
All the changes made in the 22 files look good. I just miss one change in https://github.com/IHE/DEV.SDPi/pull/290/files#diff-337f083e79d910698db486bac8af0a8d03c808712f1c07595bf8c741edcbd7f6 (see inline comment)

File: asciidoc/volume1/conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc
[the remark this answers is not in the document]
Comment (JavierEspina) [reply asciidoc/volume1/conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc:1]: Yes, I agree with that alternative solution. Easier to implement and a safer forward-looking statement :-)

[the remark this answers is not in the document]
Comment (d-gregorczyk) [reply asciidoc/volume1/conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc:1]: Damn, I am missing a changelog entry... gimme a second...

[the remark this answers is not in the document]
Comment (d-gregorczyk) [reply asciidoc/volume1/conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc:1]: There we go: https://github.com/IHE/DEV.SDPi/pull/290/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR21

**Review by JavierEspina — APPROVED** [review]

**Pull request merged**